docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,103 @@
+
+4. Select which track on the timeline you want to record onto using the \[Track\] softkey or tap on the flashing dotted area of the track you want to use. 
+You can add triggers to an existing track or create a new track by tapping on the blank area at the bottom.
+
+    ![Timeline quick record dotted tracks](/docs/images/Timeline-Quickrecord-Dotted.png)
+
+5. Tap again on the track to start recording (or press the \[Start Live Record\] softkey). The workspaces will be outlined in red but recording will not begin until the timecode is started.
+
+6. For internal timecode source, press the \{>\} (play) button in the top left of the timeline window to start the timecode running. If you are using an external timecode source, start it.
+
+7. Now when you make any of the above trigger actions on the console, you will see a trigger recording appear on the timeline. The time range you have recorded is highlighted in red on the timeline view.
+
+8. When you have finished recording press the \{O\} (record) button again and stop the timecode.
+
+You should now see all the triggers you have created laid out on the timeline.
+
+- When using internal timecode, pressing \{O\} (record) then \{>\} (play) will automatically start Live Recording.
+
+- While you are in Live Record, the console will follow the Timeline Release settings as you've programmed them in the [Timeline Options - Release tab](../timelines/timeline-options.md#release-tab), which may make playbacks behave differently to how they are programmed. A warning is displayed in the system prompt area to show you what release settings are in effect.
+
+### Auto Simplify
+
+Normally while you are recording, the \[Auto Simplify\] option is enabled. When you stop the recording, this will simplify your fader movements to a few trigger points with linear fades rather than a large number of points which are difficult to edit. If you really need to capture an exotic fader movement you can turn this option off, but to help keep your timeline easy to edit you should leave it on where you can.
+
+Trigger with multiple points during Live Record:
+
+![Timeline before auto simplify](/docs/images/Timeline-Live-Record.png)
+
+The same trigger after Auto Simplify:
+
+![Timeline after auto simplify](/docs/images/Timeline-Live-Record-Simplified.png)
+
+- When you select a trigger (by clicking on it), there are also context menu options in the \{Tools\} context menu to \{Simplify Selected Triggers\} and \{Smooth Selected Triggers\}. The Smooth option will smoothly fade between selected triggers using the fade time of the following trigger.
+
+Adding manual triggers
+-----------------
+
+If you are programming a show where you have been given a list of timecodes for each cue point, you can enter the triggers manually at those specific times.
+
+1.  Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
```

Comment:
This isn't the way we'd usually suggest people open timeline view - use `<View> <Select>` for this in preference to record

Reply:
So you have eluded to 2 ways of adding programming to a timeline in these steps. There are a few:
1. Record and selecting the timeline handle will allow you to record straight into the timeline.
2. Record and selecting any where in the time line section of the timeline view will allow you to record straight into the View.
3. Copy/Move can be used to drop an existing playback straight into the timeline.
4. Finally you can use the '+' menu if you want more choices.

Reply:
Please can you explain about the copy reference/new as I can't see how that works

Reply:
The copy reference/new is only really relevant in the add existing playback menu. I think it shows up in others currently but it shouldn't do.

All it does is if you are adding a playback that already exists you can either make the triggers act on that existing playback, or make a new copy and act on that instead.